tests/libnet/cluster.go

```diff
@@ -44,3 +47,11 @@ func getPodByKubeVirtRole(virtClient kubecli.KubevirtClient, kubevirtPodRole str
 	}
 	return &pods.Items[0], nil
 }
+
+func SkipIpv6TestsOnSingleStackCluster(virtClient kubecli.KubevirtClient) {
```

Comment:
I've only noticed this function now, sorry for that; shouldn't this test skip *if* you have only a single IPv6 address ? e.g. a cluster with a single IPv4 address is not dual stack.

Reply:
Will change to `SkipWhenNotDualStackCluster`